spec/requests/api/alerts_spec.rb

```diff
@@ -44,4 +44,101 @@
       "id"   => alert_status.id
     )
   end
+
+  context "alert_actions subcollection" do
+    let(:alert) { FactoryGirl.create(:miq_alert_status) }
+    let(:actions_subcollection_url) { "#{alerts_url(alert.id)}/alert_actions" }
+
+    it "forbids access to alerts actions subcolletion without an appropriate role" do
+      FactoryGirl.create(
+        :miq_alert_status_action,
+        :miq_alert_status => alert,
+        :user             => FactoryGirl.create(:user)
+      )
+      api_basic_authorize
+      run_get(actions_subcollection_url)
+      expect(response).to have_http_status(:forbidden)
+    end
+
+    it "reads an alert action as a sub collection under an alert" do
+      api_basic_authorize subcollection_action_identifier(:alerts, :alert_actions, :read, :get)
+      alert_action = FactoryGirl.create(
+        :miq_alert_status_action,
+        :miq_alert_status => alert,
+        :user             => FactoryGirl.create(:user)
+      )
+      run_get(actions_subcollection_url)
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to eq(
+        "name"      => "alert_actions",
+        "count"     => 1,
+        "subcount"  => 1,
+        "resources" => [
+          {
+            "href" => "http://www.example.com/api/alerts/#{alert.id}/alert_actions/#{alert_action.id}"
```

Comment:
would prefer:

```
  "resources" = [
     {
         "href" => a_string_matching("#{alerts_url(alert.id)}/alert_actions/#{alert_action.id}")
     }
```

including the ones in the alerts_spec.rb, but those can be a different PR.